dxlconsole/modules/certificates/module.py

```diff
@@ -954,8 +969,17 @@ def get(self, *args, **kwargs):
                           "ipAddress": ip_address}
                 brokers.append(broker)
 
+            # extract the WebSocket broker list from the config
+            if config_parser.has_section("BrokersWebSockets"):
+                for name, value in config_parser.items("BrokersWebSockets"):
```

Comment:
We could create a method to do this code as the same thing is also done for the "Brokers" section but it feels a little bit like overkill for this.